components/popover/popover-mixin.js

```diff
@@ -62,34 +69,107 @@ export const PopoverMixin = superclass => class extends superclass {
 
 	constructor() {
 		super();
+		this.noAutoClose = false;
 		this.opened = false;
-		this._useNativePopover = isSupported;
+		this._useNativePopover = isSupported ? 'manual' : undefined;
+		this._handleAutoCloseClick = this._handleAutoCloseClick.bind(this);
+		this._handleAutoCloseFocus = this._handleAutoCloseFocus.bind(this);
 	}
 
 	connectedCallback() {
 		super.connectedCallback();
+		if (this.opened) this._addAutoCloseHandlers();
 	}
 
 	disconnectedCallback() {
 		super.disconnectedCallback();
+		this._removeAutoCloseHandlers();
 	}
 
 	updated(changedProperties) {
 		super.updated(changedProperties);
 		if (changedProperties.has('opened')) {
 
+			if (this._useNativePopover) {
+				if (this.opened) this.showPopover();
+				else this.hidePopover();
+			}
+
+			//this._previousFocusableAncestor =
```

Comment:
I'm not sure we need this - dropdown uses this [here](https://github.com/BrightspaceUI/core/blob/main/components/dropdown/dropdown-content-mixin.js#L585) and I'm not quite sure why.

Reply:
[This](https://github.com/BrightspaceGraveyard/dropdown/pull/79) is the why it was originally there (getting there was a wild ride of polymer and lit conversions) so perhaps just testing if that case is still an issue

Reply:
Holy smokes! Thanks for finding that. I suspect this is still an issue but I will test it out.

Reply:
Ok, confirmed this indeed happens, so I will include this check.

Just in case we ever lose that history (Oct 4, 2016, Adam's change BrightspaceUI/dropdown): 
> When a dropdown has an ancestor that's focusable, that ancestor may (in all cases I've observed, will) gain focus when an empty area of a dropdown is clicked. This adds an additional check in the focus handler that the focused element isn't a focused ancestor, so that autoclose is deferred to the click handler instead as in other cases.